pyproject.toml

```diff
@@ -1,4 +1,53 @@
+[project]
+name = "QEfficient"
+dynamic = ["version"]
+description = """
+    QEfficient is the library interface for the Hugging Face Transformer \
+    models for efficient inference on Qualcomm Cloud AI 100"""
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [{ name = "Qualcomm Cloud AI ML Team" }]
+keywords = ["transformers", "Cloud AI 100", "Inference"]
+classifiers = [
+    "Programming Language :: Python :: 3", 
+    "Development Status :: 5 - Development/Unstable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Operating System :: Linux",
+    "Programming Language :: Python :: 3.8",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence for Inference Accelerator",
+]
+dependencies = [
+    "transformers==4.41.2",
+    "torch@https://download.pytorch.org/whl/cpu/torch-2.0.0%2Bcpu-cp38-cp38-linux_x86_64.whl#sha256=354f281351cddb590990089eced60f866726415f7b287db5105514aa3c5f71ca",
```

Comment:
We can add dependency `url `in the *.toml. and remove the torch installation via specific hash, just torch==2.0.0+cpu should work fine.

Reply:
Yeah, that's the right way